CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -228,6 +230,8 @@ void SiPixelLorentzAnglePCLHarvester::dqmEndJob(DQMStore::IBooker& iBooker, DQMS
   f1->SetParName(5, "quintic term");
 
   double p1_simul_newmodule = 0.294044;
+  auto width_ = phase1PixelTopology::pixelThickness;
```

Comment:
it would be better to take it really from DB as we might want to extend the functionality to phase-2...